src/std/main.ab

```diff
@@ -128,3 +128,52 @@ pub fun includes(arr, value) {
     }
     return false
 }
+
+pub fun create_symbolic_link(origin: Text, destination: Text): Bool {
+    if file_exist(origin) {
+        unsafe $ln -s "{origin}" "{path}"$
+        return true
+    }
+
+    echo "The file {$origin} doesn't exist!"
+    return false
+}
+
+pub fun create_dir(path: Text): Null {
+    if not dir_exist(origin) {
+        unsafe $mkdir -p "{path}"$
+    }
+}
+
+pub fun make_executable(path: Text): Bool {
+    if file_exist(origin) {
+        unsafe $chmod +x "{path}"$
+        return true
+    }
+
+    echo "The file {$path} doesn't exist!"
+    return false
+}
+
+pub fun make_executable(path: Text): Bool {
+    if file_exist(origin) {
+        unsafe $chmod +x "{path}"$
+        return true
+    }
+
+    echo "The file {$path} doesn't exist!"
+    return false
+}
+
+pub fun switch_user_permission(user: Text, path: Text): Bool {
+     if file_exist(origin) or dir_exist(origin) {
+        unsafe $chown -R "{user}" "{path}"$
+        return true
+    }
+
+    return false
+}
+
+pub fun download(url: Text, path: Text): Bool {
+        unsafe $wget "{url}" -P "{path}"$
```

Comment:
The `wget` command is not installed as standard in some environments.
Isn't it controversial to include it in `std`?

Reply:
i agree, this could make some serious inconsistencies between that and `curl` and `aria2c`. it would be smart to either have a function that finds which one of those is installed or just leave it out for now

Reply:
I agree I will improve it